### DISCUSSION TITLE
RabbitmqClient Can't work

### DISCUSSION THREAD

**Discussion by skyDuanXianBing** [microsoft/aspire#3997] [Q&A]
I want to develop a simple framework for using rabbitmq quickly with aspire, but I seem to be sending messages that I can't receive.
![image](https://github.com/dotnet/aspire/assets/164180639/8a7af228-49c1-4990-95a2-d15894f80f70)
![image](https://github.com/dotnet/aspire/assets/164180639/d2208cc1-c3d9-47e4-821c-140b7b235b6b)

**Comment by skyDuanXianBing** [comment]
![image](https://github.com/dotnet/aspire/assets/164180639/42800a24-e2a9-4982-938d-70ba6f5319ff)
![image](https://github.com/dotnet/aspire/assets/164180639/848d6bee-a949-4920-9009-068e8a2f43f7)

**Comment by skyDuanXianBing** [comment]
![image](https://github.com/dotnet/aspire/assets/164180639/178d7d9b-b42a-4b36-af26-0db5c0a0bb2d)

**Comment by davidfowl** [comment]
Hard to tell you why its not working from the screenshots. Can you provide a repository on github with a minimal sample? Maybe the dashboard can help you diagnose why it might not be working as well.
